docs/src/main/tut/optics/lens.md

```diff
@@ -101,6 +101,41 @@ val address = GenLens[Person](_.address)
 (address composeLens streetNumber).get(john)
 (address composeLens streetNumber).set(2)(john)
 ```
+### Other Ways of Lens Composition
+
+Is possible to compose few `Lenses` together by using `compose`:
+
+```tut:silent
+GenLens[Person](_.name).set("Mike") compose
+GenLens[Person](_.age).modify(_ + 1)
+```
+
+Same but with the simplified macro based syntax:
+
+```tut:silent
+import monocle.macros.syntax.lens.
+
+john
+  .lens(_.name).set("Mike") compose
+  .lens(_.age).modify(_ + 1)
+```
+
+Sometimes you need an easy way to update `Product` type inside
```

Comment:
Maybe worth mentioning that a more detailed view of the various optics composition is available here: http://julien-truffaut.github.io/Monocle/optics.html